servers/fastify/session/https-session-server-redis/https-session-server-redis.js

```diff
@@ -41,29 +34,38 @@ const fastify = Fastify({
   logger: true
 })
 
-fastify.register(fastifyCookie)
+const initialization = async () => {
```

Comment:
if you pass in the fastify app as a parameter, you can export this function and make this easily testable. I would also move the listen outside.

```suggestion
const initialization = async (app) => {
```